torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -804,6 +827,11 @@ class TensorExprFuser {
 
     // Now all the nodes that we're going to fuse are moved next to the fusion
     // group, so we can safely merge them into the fusion group subgraph.
+
+    fusion_group = MakeOutplaceNode(fusion_group, true);
+    to_merge = MakeOutplaceNode(to_merge, true);
+    nodes_to_merge[0] = to_merge;
```

Comment:
This doesn't look right. Why is it only for `nodes_to_merge[0]`? What about other merged nodes? Also, the new code separates the comment from the existing code, which the comment refers to.

Reply:
Hi, thanks for pointing out this.
From my understanding, "nodes_to_merge" will only have 1 node except "cat".https://github.com/pytorch/pytorch/blob/f91c1f9180ef0b01eddf682e9c9190d9a392ded6/torch/csrc/jit/passes/tensorexpr_fuser.cpp#L809-L815.
 And for the "cat" case, both "cat" and "list construct" do not belong to the "inplace node", we will do nothing for them.